### PR TITLE
Add convenience functions to remove build images and containers

### PR DESCRIPTION
**Pull request by h44z**

### CODE REVIEW

**Comment by fbartels** [comment]
The clean all images does not actually remove all images:

```
$ docker images --format '{{.Repository}}:{{.Tag}}' | grep 'fbartels/kopano_'
fbartels/kopano_base:<none>
```
from a quick search there does not really to really delete all these images in a single command.

What I am using when I want to clean out my local system is `docker system prune -a`

**Comment by h44z** [comment]
Hm, is the base image used in any container? If I try this call, all build images get deleted.

**Comment by fbartels** [comment]
Yes, kopano_base is included as the base for for kopano_core, kopano_php, kopno_zpush, .. basically everything Debian based.

It feels like Docker just occasionally creates these dangling images. Pruning my system and then running `make build-all` already creates this:

```
$ docker image ls --filter dangling=true
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
<none>              <none>              dedb46d1f03d        11 minutes ago      22.8MB
<none>              <none>              7b56fd2a2d73        15 minutes ago      376MB
```

But at least none with the "none" tag.

File: Makefile [hunk 1]
```diff
@@ -358,6 +358,14 @@ clean:
 	docker ps --filter name=kopano_test* -aq | xargs docker rm -f || true
 	docker-compose -f $(DOCKERCOMPOSE_FILE) down -v --remove-orphans || true
 
+.PHONY: clean-all-images
+clean-all-images:
+	docker rmi $$(docker images --format '{{.Repository}}:{{.Tag}}' | grep '${docker_repo}/kopano_')
```
Comment (fbartels) [inline Makefile]: adding a `| grep -v '<none>'` will skip all items with the tag that cannot be deleted

**Review by fbartels — CHANGES_REQUESTED** [review]
I think with the inline change it would be good to merge

**Comment by fbartels** [comment]
Thanks!

**Pull request merged**